src/Plugin/EntityBrowser/WidgetValidation/EntityType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity_browser\Plugin\EntityBrowser\WidgetValidation\EntityType.
+ */
+
+namespace Drupal\entity_browser\Plugin\EntityBrowser\WidgetValidation;
+
+use Drupal\entity_browser\WidgetValidationBase;
+use Symfony\Component\Validator\ConstraintViolationList;
+
+/**
+ * Validates that the widget returns the appropriate number of elements.
+ *
+ * @EntityBrowserWidgetValidation(
+ *   id = "entity_type",
+ *   label = @Translation("Entity type validator"),
+ * )
+ */
+class EntityType extends WidgetValidationBase {
+  /**
+   * {@inheritdoc}
+   */
+  public function validate(array $entities, $options = []) {
```

Comment:
Body of validate() in both plugins look very similar. Would be great if we could move more common logic to the base class.